core/include/traccc/utils/seed_generator.hpp

```diff
@@ -76,8 +72,10 @@ struct seed_generator {
 
         for (std::size_t i = 0; i < e_bound_size; i++) {
 
-            bound_param[i] = std::normal_distribution<scalar>(
-                bound_param[i], m_stddevs[i])(m_generator);
+            if (m_stddevs[i] != scalar{0}) {
+                bound_param[i] = std::normal_distribution<scalar>(
+                    bound_param[i], m_stddevs[i])(m_generator);
+            }
```

Comment:
Shouldn't this still set the parameter to the mean if the deviation is zero?

Reply:
Yes, but I think the mean is `bound_param[i]`, so in this case nothing needs to be done. Or am I reading this wrong?